main/broadcasters.c

```diff
@@ -389,12 +388,78 @@ static broadcaster_ops_t mijia_temp_hum_ops = {
     .is_broadcaster = mijia_temp_hum_is_broadcaster,
     .metadata_get = mijia_temp_hum_metadata_get,
 };
+/* Beewi Smart Door */
+#define BEEWI_SMART_DOOR_COMPANY_ID     0x000D
+#define BEEWI_SMART_DOOR_SERVICE_ID     0x08 
+#define BEEWI_SMART_DOOR_DATA_TYPE_STAT 0x0C
+#define BEEWI_SMART_DOOR_DATA_TYPE_BATT 0x06
+
+
+typedef struct {
+    uint16_t company_id;
+    uint8_t  service_id;
+	uint8_t  type_stat;
```

Comment:
You have here and a few other places inconsistent indentation. Could you please make sure to use only spaces and remove tabs? (Yes, I'm one of those guys...)